### PR TITLE
Add supervisor_ingress support

### PR DESCRIPTION
**Pull request by ludeeus**
Frontend: https://github.com/home-assistant/frontend/pull/9087

### CODE REVIEW

File: redirect.json [hunk 1]
```diff
@@ -233,6 +233,19 @@
     "description": "show your Supervisor system information",
     "introduced": "supervisor-2021.02.12"
   },
+  {
+    "redirect": "supervisor_ingress",
+    "name": "Supervisor: Ingress",
+    "badge": "Ingress",
+    "description": "Open the ingress URL of a Supervisor add-on",
```
[the remark this answers is not in the document]
Comment (frenck) [reply redirect.json:240]: Open web UI can also be something else (non-Ingress).

[the remark this answers is not in the document]
Comment (bramkragten) [reply redirect.json:240]: I know, but is that important for the user?

[the remark this answers is not in the document]
Comment (frenck) [reply redirect.json:240]: No, but I assume those won't work?

Ingress is a user facing term.

[the remark this answers is not in the document]
Comment (ludeeus) [reply redirect.json:240]: "Ingress" is what we use to describe this feature everywhere, I think it's fine to use that

[the remark this answers is not in the document]
Comment (bramkragten) [reply redirect.json:240]: OK :-)

File: redirect.json [hunk 2]
```diff
@@ -233,6 +233,19 @@
     "description": "show your Supervisor system information",
     "introduced": "supervisor-2021.02.12"
   },
+  {
+    "redirect": "supervisor_ingress",
+    "name": "Supervisor: Ingress",
+    "badge": "Ingress",
```
Comment (bramkragten) [inline redirect.json:239]: Or maybe just this one, `Add-on: Ingress`

File: redirect.json [hunk 3]
```diff
@@ -1,4 +1,17 @@
 [
+  {
+    "redirect": "supervisor_ingress",
+    "name": "Add-on: Ingress",
+    "badge": "Ingress",
```
Comment (bramkragten) [inline redirect.json:5]: You changed the name now, and not the badge, intentional?

Comment (bramkragten) [reply redirect.json:5]: yes 😄

**Pull request merged**